### PR TITLE
config: add Fedora 35 configs

### PR DESCRIPTION
**Pull request by eclipseo**
Close: #762

### CODE REVIEW

**Comment by sergiomb2** [comment]
Hi, as in https://github.com/rpm-software-management/mock/commit/3480b06be2ab86133ade765619ffd3b21d7e7d60 we and bump rawhide release to 36 in mock-core-configs/etc/mock/templates/fedora-rawhide.tpl

**Comment by hroncok** [comment]
The CI failed only because it tries to build the package in Copr and Rawhide Copr chroots are now broken by the thing that this fixes.

**Comment by hroncok** [comment]
This fixes the 35 roots for me, but rawhide/36 still fails.

**Comment by sergiomb2** [comment]
for a quick fix, we need update
/etc/mock/templates/fedora-rawhide.tpl line 12 with
config_opts['releasever'] = '36', as reference https://github.com/rpm-software-management/mock/commit/3480b06be2ab86133ade765619ffd3b21d7e7d60#diff-6d85f049d29a9ebeda1e02d48fb369982d6410ba63c64306001e1f41f8a3c4dd

**Comment by Conan-Kudo** [comment]
Please drop the tag-release commit in here, merging that won't bring the tag over into the main repository, and it blocks @praiskup from releasing mock-core-configs-35.

File: .tito/packages/mock-core-configs [hunk 1]
```diff
@@ -1 +1 @@
-34.6-1 mock-core-configs/
+35-1 mock-core-configs/
```
Comment (xsuchy) [inline .tito/packages/mock-core-configs:1]: This should not be part of this change. This is done automatically when you do `tito tag` which is part of release process.

File: mock-core-configs/mock-core-configs.spec [hunk 2]
```diff
@@ -149,6 +149,9 @@ fi
 %ghost %config(noreplace,missingok) %{_sysconfdir}/mock/default.cfg
 
 %changelog
+* Sat Aug 14 2021 Robert-André Mauchin <zebob.m@gmail.com> 35-1
+- Add Fedora 35 configs
+
```
Comment (xsuchy) [inline mock-core-configs/mock-core-configs.spec:154]: This as well. The changes in the spec file are done during release process.

**Comment by praiskup** [comment]
Thank you for the PR, I'm taking the first commit (plus 35=>36 change in rawhide template).

**Comment by eclipseo** [comment]
Sorry I was working, didn't have time to follow up as quickly as expected.
